tokio-threadpool/src/builder.rs

```diff
@@ -138,6 +145,33 @@ impl Builder {
         self
     }
 
+    /// Set the maximum number of concurrent blocking sections.
+    ///
+    /// This must be a number between 1 and 32,768 though it is advised to keep
+    /// this value on the smaller side.
```

Comment:
What happens when the number of `blocking` calls exceeds `max_blocking`? Should be called out here.